portfolio/src/main/webapp/slide.css

```diff
@@ -5,6 +5,10 @@
 
 .pic {
   display: none;
+}
+
+.pic-show {
+  display: block;
   margin: auto;
```

Comment:
If the goal of this style is just to toggle the display, remove the "margin: auto" field into a separate style. This way you won't accidentally change margins when reusing the style to hide and show elements.